recipes/sirius/v7.5/gh200/config.yaml

```diff
@@ -0,0 +1,7 @@
+name: sirius-gh200
+store: /user-environment
+description: The latest version of Sirius (v7.5.0) and the tools for building it, configured for GH200.
+spack:
+  repo: https://github.com/spack/spack.git
+  # commit on develop that updated the sirius package.py for v7.5.0 of sirius
+  commit: 13b711f
```

Comment:
Is it possible to use a released version of Spack (`0.22.0` is now out), and add the SIRIUS package in the `repo/` folder if needed? I think this is the preferred way to do things.